### PR TITLE
updating readme

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
@joshmoore could you confirm that works better?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,14 +38,15 @@ Install the latest version of OMERO.server and OMERO.web and then:
     $ wget -O master.zip https://github.com/openmicroscopy/virtual-microscope/zipball/master
     $ unzip master.zip
     $ mv openmicroscopy-virtual-microscope-* virtual-microscope
+    $ ln -s /path/to/virtual-microscope /path/to/omero/lib/python/omeroweb/virtualmicroscope
```
Comment (joshmoore) [inline README.md]: Based on https://github.com/openmicroscopy/virtual-microscope/pull/3, I would remove this and just say something about having it on PYTHONPATH.

Comment (aleksandra-tarkowska) [inline README.md]: actually we don't need to add anything as we are already in omeroweb

**Comment by will-moore** [comment]
@aleksandra-tarkowska The way we have done all the other OMERO.web apps is to say they should be in a directory on your PYTHONPATH, not in omeroweb/.

You now have the app named 'virtualmicroscope' under production install and virtual-microscope in development install.

**Comment by will-moore** [comment]
Following your develop install, when I try to start web I get:
```
ImportError: No module named virtual-microscope
```

**Comment by aleksandra-tarkowska** [comment]
@will-moore check latest update Python doesn't allow ```-```

**Comment by aleksandra-tarkowska** [comment]
@will-moore you may want to update https://github.com/openmicroscopy/webemdb/blob/master/README.md as I copied from there

**Pull request merged**